README.md

```diff
@@ -333,6 +333,12 @@ VM
 : Virtual machine ran in KVM. Pennyworth supports running VMs described in a
 Vagrantfile as well as non vagrant managed ones.
 
+## Release Cycle
+
+As for now we don't have a time based release cycle. We update and publish the
+gem as needed. However don't hesitate about submitting bugs or better yet pull
```

Comment:
I'd rather phrase this as follows:
`However don't hesitate to report bugs or better yet submit fixes as pull requests.`